### DISCUSSION TITLE
Change apps (aka Window Switcher, Alt+Tab, etc...) not working after an update (Window 11)

### DISCUSSION THREAD

**Discussion by Laytonary** [valinet/ExplorerPatcher#1830] [General]
Yesterday I was updating somethings from window updater, and some things happen:
First: Change apps not working anymore, no matter how I press alt+tab or win+tab or 3 fingers gesture on touchpad, it's not working, at first, I thought it was from window, but I ask every forum about window (also laptop's manufacturer website). But after thinking about the problem again, I knew this was because of Explorer Patcher, is there anything to fix this or do I have to wait for the next EP update?
Second: On-screen popup also disappears, as you know, on window, every time you change brightness or volume by keyboard (Fn+F(x)) it will popup an on-screen window immediately, but after update, it doesn't popup anymore

I hope the devs could fix this as soon as possible cause those really important to my workflow
Last but not least, hope all the devs who making this software all the best cause I really love this software man, really appreciate you devs!!!!

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Please close this one.
